examples/basics/visuals/custom_visual.py

```diff
@@ -68,29 +68,29 @@ class MarkerVisual(Visual):
             }
         }
     """
-    
+
     def __init__(self, pos=None, color=None, size=None):
-        self._program = ModularProgram(self.VERTEX_SHADER, 
+        self._program = ModularProgram(self.VERTEX_SHADER,
                                        self.FRAGMENT_SHADER)
         self.set_data(pos=pos, color=color, size=size)
-        
+
     def set_options(self):
         """Special function that is used to set the options. Automatically
         called at initialization."""
-        gloo.set_state(clear_color=(1, 1, 1, 1), blend=True, 
+        gloo.set_state(clear_color=(1, 1, 1, 1), blend=True,
```

Comment:
@almarklein, how will the current gloo/glir know to which context this call belongs? Does this need to be changed to `self.set_gl_state(...)`?